transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java

```diff
@@ -829,4 +810,25 @@ Entry recycleAndGetNext() {
             return next;
         }
     }
+
+    private static final class ChannelWritabilityChangedTask implements Runnable {
+        private final Channel channel;
+        private boolean writable = true;
```

Comment:
that is true but I think we always fire the first writability event when moving to "non-writable" or I'm wrong ?